manipulators/gripper_interface/scripts/auto_gripper_interface_node.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python3
+
+import rospy
+from std_msgs.msg import Bool
+from time import sleep
+from datetime import datetime
+import Jetson.GPIO as GPIO
+
+# Note: is currently hardware dependent
+
+on = -1.0
+off = 1.0
+active = 1
+inactive = 0
+

```

Comment:
```suggestion

ON = -1.0
OFF = 1.0
ACTIVE = 1
INACTIVE = 0
```

since these are constants defined outside any class